packages/w-components/components/spa/SPAPage.js

```diff
@@ -82,4 +83,9 @@ class SPAPage extends WComponent{
   }
 }
 SPAPage.prototype.stylesheet=stylesheet;
+
+const BASE_NAME = window.location.pathname.substring(0, window.location.pathname.lastIndexOf("/"));
```

Comment:
Add this basename setting temporary for avoiding exception caused by wconfig not set.